openrest4j-api/src/main/java/com/openrest/olo/dispatches/PickupDispatch.java

```diff
@@ -29,6 +35,14 @@ protected PickupDispatch cloneImpl() {
         return new PickupDispatch(
                 (time != null) ? (Date) time.clone() : null,
                 timeGuarantee, charge,
-                ((properties != null) ? new LinkedHashMap<>(properties) : null));
+                ((properties != null) ? new LinkedHashMap<>(properties) : null),
+                curbsideAdditionalInformation,
+                isCurbside);
     }
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public Boolean isCurbside;

```

Comment:
Please = false here instead of constructor

Reply:
done